CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased Version
 - Renamed `providers` to `authProviders` in project collection **Breaking Change**
+- Support for Backblaze adapter in Storage
+- Support for Linode adapter in Storage
```

Comment:
This should go into 0.14.2 section probably. Let's sync with master branch, and update it.